src/components/NavigationListAccordion.tsx

```diff
@@ -46,7 +46,7 @@ export const NavigationListAccordion = ({
             display: 'flex',
             gap: '1rem',
             alignItems: 'center',
-            'svg, img, div': { borderRadius: '50%', height: '25px', width: '25px' },
```

Comment:
pass på at dette ikke brekker noe annet da! Kan godt være at det er annet ræl som ikke trengs lenger, men kan være at det er der for en grunn også. `NavigationListAccordion` brukes jo ganske mange plasser, feks er ikonene på søk også endret nå, vet ikke om det er med hensikt?

![bilde](https://github.com/BIBSYSDEV/NVA-Frontend/assets/6313468/a669eba1-d279-44ec-847c-bb4f71d1d6d7)

Reply:
Her hadde visst svg sneker seg bort ja..

Reply:
img er fortsatt fjernet ser jeg, er det bevisst? antar det ble brukt for personer som har lastet opp bilde 🤔 
Bare husk å gå gjennom egen PR og spør deg selv hvorfor det og det er lagt til, fjernet og endret, så finner man mye av dette selv også :) slipper mye round-trips med QA 👍

Reply:
Blei en blunder der ja